src/common/string-builder.ts

```diff
@@ -81,8 +82,39 @@ class StringBuilder {
         return this.printQueue.join(' ');
     }
 
-   combineURLs(baseURL: string, relativeURL?: string): string {
-        return relativeURL ? new URL(relativeURL, baseURL || undefined).toString() : baseURL;
+    combineURLs(baseURL: string, relativeURL?: string): string {
```

Comment:
Looking at the test code, it doesn't seem good to use the name relativeURL anymore. In fact, there are cases where absolute paths come in, not relative paths. It would be better to change the variable name.